### DISCUSSION TITLE
Limit the number of weeks displayed in the date picker?

### DISCUSSION THREAD

**Discussion by thatmaxplayle** [gpbl/react-day-picker#2554] [Support]
Is there a way to limit the amount of weeks displayed by the `<DayPicker/>`? 

For instance, only show 2 weeks by specifying `numberOfMonths={0.5}` - unless you were to add a new prop?

Cheers!

**Comment by gpbl** [comment]
@thatmaxplayle, try using a `Week` [custom component](https://daypicker.dev/guides/custom-components) that doesn't render anything if you need to hide the week row.
